GraphsV2/utils/helper.js

```diff
@@ -391,10 +391,4 @@ export const wrapD3Text = (text, width) => {
   });
 };
 
-export const insertTimestampToTooltip = ({ tooltip, xColumn }) => {
```

Comment:
let's keep this function and create a new export const insertElementIntoTooltip = (tooltip, element) => ([element, ...tooltip]); and use here.